tests/lib/Validator/ResourceTypeValidatorTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\Tests\Validator;
+
+use Netgen\Layouts\Sylius\Validator\Constraint\ResourceType;
+use Netgen\Layouts\Sylius\Validator\ResourceTypeValidator;
+use Netgen\Layouts\Tests\TestCase\ValidatorTestCase;
+use PHPUnit\Framework\Attributes\CoversClass;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\ConstraintValidatorInterface;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+#[CoversClass(ResourceTypeValidator::class)]
+final class ResourceTypeValidatorTest extends ValidatorTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->constraint = new ResourceType();
+    }
+
+    public function getValidator(): ConstraintValidatorInterface
+    {
+        $allowedResources = [
+            'Sylius\Component\Product\Model\ProductInterface' => 'product',
```

Comment:
`::class` constants.

Reply:
Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/11/commits/93ab79cc5e0f027f9222331aa0551e4358fa80db